scripts/ops/env.dist

```diff
@@ -0,0 +1,11 @@
+# copy this file to ie "/etc/armadillo/acc.env"
```

Comment:
Why do you suggest another path than in the `.README`? I suggest keeping it the same.

Reply:
Thanks ... done